src/async-calendar.js

```diff
@@ -0,0 +1,53 @@
+const assignments =
+    'div.fc-event';
+const intervalTimer = 200;
+
+document.styleSheets[0].insertRule(":root{--comp-color: #000000; --cal-border-color: #000000; --cal-text-color: #000000}");
+
+setInterval(() => {
+    updateColor();
+}, 500);
+
+
+// jQuery Document Ready
+$(function () {
+    console.log('jQuery Ready...');
+    setTimeout(assignmentsInterval, intervalTimer);
+});
+
+const assignmentsInterval = async () => {
+    if ($(assignments).length > 0) {
+        await getStorage();
+        return updateCalendar();
+    }
+    setTimeout(assignmentsInterval, intervalTimer);
+};
+
+const updateCalendar = function () {
+    let completedElements = [];
```

Comment:
```suggestion
    const completedElements = [];
```
you are not specifically assigning another value so you can change this to const.